### PR TITLE
Update `puppetlabs/stdlib` dependency to allow 6.x and require at least 4.19.0 (where the `fact()` function was introduced)

### PR DESCRIPTION
**Pull request by pillarsdotnet**
Fixes: #458

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 < 6.0.0"
+      "version_requirement": ">= 4.19 < 6.0.0"
```
Comment (bastelfreak) [inline metadata.json]: please also provide the patch level:
```suggestion
      "version_requirement": ">= 4.19.0 < 6.0.0"
```

Comment (alexjfisher) [reply metadata.json]: stdlib 6.0.0 was released 5 hours ago.

Comment (alexjfisher) [reply metadata.json]:
```suggestion
      "version_requirement": ">= 4.19.0 < 7.0.0"
```

Comment (pillarsdotnet) [reply metadata.json]: Changed as requested.

**Comment by Dan33l** [comment]
@pillarsdotnet please squash the 2 commits into one.

**Comment by pillarsdotnet** [comment]
As requested.

**Comment by alexjfisher** [comment]
@pillarsdotnet Thanks!

**Pull request merged**